### PR TITLE
Add missing code to anomaly detection phone call tutorial

### LINKED ISSUE #23888: No anomalies detected
In the sample for phone calls anomaly detection we noticed that you won’t get any anomalies detected with version 5.5. Running the sample with version 5.3 works.

Not sure if this is an intended change in the library requiring an update of sample data or if this is due to a problem in the TimeSeries part?

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5e6266e1-f8e5-d303-7ddc-be2d6e47f831
* Version Independent ID: 2ae35df3-cf3a-ed1c-b466-a8be8a9729c3
* Content: [Tutorial: Detect anomalies in phone calls - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/phone-calls-anomaly-detection)
* Content Source: [docs/machine-learning/tutorials/phone-calls-anomaly-detection.md](https://github.com/dotnet/docs/blob/main/docs/machine-learning/tutorials/phone-calls-anomaly-detection.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### LINKED ISSUE #25363: DetectPeriod method code should return the period value
The code in the [Create the DetectPeriod method](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/phone-calls-anomaly-detection#create-the-detectperiod-method) section has a bug. The method `DetectPeriod` should have a return type of `int` and should return the period value.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 5e6266e1-f8e5-d303-7ddc-be2d6e47f831
* Version Independent ID: 2ae35df3-cf3a-ed1c-b466-a8be8a9729c3
* Content: [Tutorial: Detect anomalies in phone calls - ML.NET](https://docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/phone-calls-anomaly-detection)
* Content Source: [docs/machine-learning/tutorials/phone-calls-anomaly-detection.md](https://github.com/dotnet/docs/blob/main/docs/machine-learning/tutorials/phone-calls-anomaly-detection.md)
* Product: **dotnet-ml**
* GitHub Login: @luisquintanilla
* Microsoft Alias: **luquinta**

### PR DESCRIPTION
**Pull request by luisquintanilla**
- Update `DetectPeriod` signature
- Add missing code snippet to return `period`.
- Specify package versions (1.5.2).

Fixes #25363
Fixes #23888 

[Preview Link](https://review.docs.microsoft.com/en-us/dotnet/machine-learning/tutorials/phone-calls-anomaly-detection?branch=pr-en-us-25384)

AB#1864062

### CODE REVIEW

**Review by jwood803 — APPROVED** [review]

File: docs/machine-learning/tutorials/phone-calls-anomaly-detection.md [hunk 1]
```diff
@@ -32,11 +32,9 @@ You can find the source code for this tutorial at the [dotnet/samples](https://g
...
 
-3. Install the **Microsoft.ML NuGet Package**:
+3. Install the **Microsoft.ML NuGet Package** version **1.5.2**:
 
-    [!INCLUDE [mlnet-current-nuget-version](../../../includes/mlnet-current-nuget-version.md)]
-
-    In Solution Explorer, right-click on your project and select **Manage NuGet Packages**. Choose "nuget.org" as the Package source, select the Browse tab, search for **Microsoft.ML** and select the **Install** button. Select the **OK** button on the **Preview Changes** dialog and then select the **I Accept** button on the **License Acceptance** dialog if you agree with the license terms for the packages listed. Repeat these steps for **Microsoft.ML.TimeSeries**.
+    In Solution Explorer, right-click on your project and select **Manage NuGet Packages**. Choose "nuget.org" as the Package source, select the Browse tab, search for **Microsoft.ML** and select the **Install** button. Select the **OK** button on the **Preview Changes** dialog and then select the **I Accept** button on the **License Acceptance** dialog if you agree with the license terms for the packages listed. Repeat these steps for **Microsoft.ML.TimeSeries** version **1.5.2**.
```
Comment (briacht) [inline docs/machine-learning/tutorials/phone-calls-anomaly-detection.md:37]: Should you add a note about selecting the version here in UI?

**Review by briacht — APPROVED** [review]

**Review by IEvangelist — APPROVED** [review]
Looks good to me, :shipit: when you're ready.

**Comment by IEvangelist** [comment]
@luisquintanilla feel free to merge this when you're ready.

File: docs/machine-learning/tutorials/phone-calls-anomaly-detection.md (hunk 1 above)
Comment (luisquintanilla) [reply docs/machine-learning/tutorials/phone-calls-anomaly-detection.md]: Done

**Pull request merged**